### PR TITLE
tmpfiles: fix documentation about quoting the "argument" field in tmpfiles.d snippets

### LINKED ISSUE #17740: systemd-tmpfilesd not processing quotes like the documentation implies
**systemd version the issue has been seen with**
 > systemd 246 (246.6-4)
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +ZSTD +SECCOMP +BLKID +ELFUTILS +KMOD +IDN2 -IDN +PCRE2 default-hierarchy=hybrid

**Used distribution**
 > Debian Testing/Bullseye x86_64

**Linux kernel version used** (`uname -a`)
 > Linux debian 5.9.0-3-amd64 #1 SMP Debian 5.9.9-1 (2020-11-19) x86_64 GNU/Linux

**CPU architecture issue was seen on**
 > x86_64

**Expected behaviour you didn't see**

To quote the [documentation](https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html):

```
The configuration format is one line per path containing type, path, mode, ownership, age, and argument fields:

#Type Path        Mode User Group Age Argument
d     /run/user   0755 root root  10d -
L     /tmp/foobar -    -    -     -   /dev/null
Fields may be enclosed within quotes and contain C-style escapes.
```

What I tried was testing quotes (double-quotes, but single-quotes behave similar),
the documentation impllies that any field could be quoted. Use a tmpfile entry like this:

```
f+ '/var/example 2' - - - - "%B\n%o\n%w\n%W%% p\n"
```

This should result (on my system) with a file liek this:

```

debian

% p

```

**Unexpected behaviour you saw**

The file replicated the argument without resolving quotes:

```
"
debian

% p
"
```

**Steps to reproduce the problem**
 
should be apparent from the points above.

I am not sure if the error lies in the documentation or implementation,
but it should be fixed to be in sync.

### PR DESCRIPTION
**Pull request by poettering**
Fixes #17740

### CODE REVIEW

File: man/tmpfiles.d.xml [hunk 1]
```diff
@@ -145,14 +145,17 @@ A+    /path-or-glob/to/append/acls/recursively -    -    -     -           POSIX
...
 
-    <programlisting>#Type Path        Mode User Group Age Argument
+    <programlisting>#Type Path        Mode User Group Age Argument…
 d     /run/user   0755 root root  10d -
 L     /tmp/foobar -    -    -     -   /dev/null</programlisting>
 
-    <para>Fields may be enclosed within quotes and contain C-style escapes.</para>
+    <para>Fields may contain C-style escapes. With the exception of the seventh field (the "argument") all
+    fields may be enclosed in quotes. Note that any whitespace found in the line after the beginning of the
+    argument field will be considered part of the argument field. To begin the argument field whith a
```
Comment (yuwata) [inline man/tmpfiles.d.xml]: whith -> with?

**Review by yuwata — CHANGES_REQUESTED** [review]
LGTM. But one typo.

File: man/tmpfiles.d.xml (hunk 1 above)
Comment (poettering) [reply man/tmpfiles.d.xml]: true

**Comment by poettering** [comment]
Fixed the typo, upgrading label.

**Pull request merged**